utils/docker/Dockerfile.centos.7

```diff
@@ -10,7 +10,7 @@ FROM centos:centos7.9.2009
 LABEL maintainer="daos@daos.groups.io"
 
 # Intermittent cache-bust.  Used to reduce load on the actual CACHEBUST later.
-ARG CB0
+ARG CB_WEEKLY
```

Comment:
Rebuilding the entire image at this interval seems a bit big-hammer-ish.

Wouldn't it be more efficient to simply do an upgrade at this interval somewhere after line 43?  I think that would make this periodic activity smaller since it's not re-installing everything that didn't have any updates since the last periodic rebuild.

Reply:
This will only happen weekly so that doesn't seem a major problem, and we could probably make it go though our proxy.

If you build from scratch on a new system them there are already three calls to "yum upgrade", yes we could add a forth and it would allow better re-use but it would also slow down other use cases where images are cleared more often.

Reply:
I would think you will still only have 3 `yum upgrade`s as you would move the one from line 20 until after your (moved lower) `ARG CB0`.

Reply:
It seems odd to do yum install ... without a yum upgrade in the same step, and it would still mean four instances of downloading the packages list which is what takes the time/bandwidth.